### DISCUSSION TITLE
difficulty using files in the public folder

### DISCUSSION THREAD

**Discussion by uisam00** [brocoders/extensive-react-boilerplate#370] [Q&A] [answered]
I would like a tutorial on how to use a file from the public folder, such as an image, just referencing it doesn't work, maybe there is some configuration missing.

**Comment by uisam00** [comment]
@sars @websumy @Shchepotin @Insidexa

**Reply by Shchepotin** [reply, marked as answer]
There was a bug. Please check the fix here: #372

Please apply it and use files from the public folder as usual.
